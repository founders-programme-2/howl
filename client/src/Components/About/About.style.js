import styled from 'styled-components';
import responsiveSizes from '../../constants/responsiveSizes';

const { SML, MDM } = responsiveSizes;

const Simg = styled.img`
  border-radius: 20px;
  max-width: 30%;
  float: right;
  margin: 0 2rem 2rem;

  @media (max-width: ${MDM}) {
    max-width: 50%;
    margin: 2rem;
  }

  @media (max-width: ${SML}) {
    max-width: 50%;
    margin: 2rem;
  }
`;

export default Simg;
