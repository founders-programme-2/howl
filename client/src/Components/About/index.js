import React, { Fragment } from 'react';
import aboutImage from '../../assets/about_page.jpg';
import Simg from './About.style';
import Footer from '../Common/Footer';

const About = ({ setFilters }) => {
  return (
    <Fragment>
      <main>
        <header>
          <h1>About Howl</h1>
        </header>
        <Simg src={aboutImage} alt="HOWL women at a protest" />
        <p>
          The History of Women&#39;s Liberation (HOWL) project arose initially
          out of discussions older/old feminists were holding about the
          importance of collecting and producing our own history. Our time is
          limited. We need to act now. In HOWL, we intend to collectively create
          a broad picture that includes and represents women from all parts of
          the UK. Our core collection years are from 1965 to the 1990s. We will
          achieve this through reaching out to old feminists, creating a
          website, collecting oral histories, gathering memorabilia and holding
          events. HOWL originated in London but we are committed to making sure
          that the voices and materials collected are diverse, inclusive and
          cover the whole of the UK.
        </p>
        <h2>Want to get involved?</h2>
        <p>
          Send an email introducing yourself to howl@howl-uk.org and come to
          one of our meetings!
        </p>
        <h2>Donate</h2>
        <p>Passionate about our cause? We are always looking for donations.</p>
        <h2>Contact Us</h2>
        <p>Have a question? Send us an email at howl@howl-uk.org</p>
      </main>
      <Footer setFilters={setFilters} />
    </Fragment>
  );
};

export default About;
