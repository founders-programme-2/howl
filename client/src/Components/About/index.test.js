import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import About from './index';

test('Natural', () => {
  const about = renderer
    .create(
      <BrowserRouter>
        {' '}
        <About />{' '}
      </BrowserRouter>
    )
    .toJSON();
  expect(about).toMatchSnapshot();
});
