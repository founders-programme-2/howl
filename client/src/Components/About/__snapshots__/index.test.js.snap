// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Natural 1`] = `
Array [
  " ",
  <main>
    <header>
      <h1>
        About Howl
      </h1>
    </header>
    <img
      alt="HOWL women at a protest"
      className="sc-bdVaJa bScuXf"
      src="about_page.jpg"
    />
    <p>
      The History of Women's Liberation (HOWL) project arose initially out of discussions older/old feminists were holding about the importance of collecting and producing our own history. Our time is limited. We need to act now. In HOWL, we intend to collectively create a broad picture that includes and represents women from all parts of the UK. Our core collection years are from 1965 to the 1990s. We will achieve this through reaching out to old feminists, creating a website, collecting oral histories, gathering memorabilia and holding events. HOWL originated in London but we are committed to making sure that the voices and materials collected are diverse, inclusive and cover the whole of the UK.
    </p>
    <h2>
      Want to get involved?
    </h2>
    <p>
      Send an email introducing yourself to fakeemail@gmail.com and come to one of our meetings!
    </p>
    <h2>
      Donate
    </h2>
    <p>
      Passionate about our cause? We are always looking for donations.
    </p>
    <h2>
      Contact Us
    </h2>
    <p>
      Have a question? Send us an email at fakeemail@gmail.com.
    </p>
  </main>,
  <footer
    className="sc-bwzfXH bmqgFz"
  >
    <section
      className="sc-htpNat jvwxwt"
    >
      <h2
        className="sc-bxivhb hOulHU"
      >
        Search Archive
      </h2>
      <button
        className="MuiButtonBase-root MuiButton-root Footer-howToSearchButton-1 MuiButton-text"
        disabled={false}
        onBlur={[Function]}
        onClick={[Function]}
        onFocus={[Function]}
        onKeyDown={[Function]}
        onKeyUp={[Function]}
        onMouseDown={[Function]}
        onMouseLeave={[Function]}
        onMouseUp={[Function]}
        onTouchEnd={[Function]}
        onTouchMove={[Function]}
        onTouchStart={[Function]}
        tabIndex={0}
        type="button"
      >
        <span
          className="MuiButton-label"
        >
          How to search
        </span>
      </button>
    </section>
    <section
      className="sc-ifAKCX dIzAIk"
    >
      <div
        className="MuiFormControl-root Footer-formControl-2"
      >
        <div
          aria-label="category"
          className="MuiInputBase-root WithStyles(ForwardRef(InputBase))-root-44 MuiInputBase-formControl Footer-select-7"
          onClick={[Function]}
        >
          <div
            className="MuiSelect-root"
          >
            <div
              aria-haspopup="true"
              aria-pressed="false"
              className="MuiSelect-select MuiSelect-selectMenu MuiInputBase-input WithStyles(ForwardRef(InputBase))-input-45"
              id="select-category"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="button"
              tabIndex={0}
            >
              <span
                className="sc-gzVnrw grgfgk"
              >
                Category
              </span>
            </div>
            <input
              id="category-customized-select"
              name="category"
              type="hidden"
              value="Category"
            />
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root MuiSelect-icon"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M7 10l5 5 5-5z"
              />
            </svg>
          </div>
        </div>
        <div
          aria-label="location"
          className="MuiInputBase-root WithStyles(ForwardRef(InputBase))-root-44 MuiInputBase-formControl Footer-select-7"
          onClick={[Function]}
        >
          <div
            className="MuiSelect-root"
          >
            <div
              aria-haspopup="true"
              aria-pressed="false"
              className="MuiSelect-select MuiSelect-selectMenu MuiInputBase-input WithStyles(ForwardRef(InputBase))-input-45"
              id="select-location"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="button"
              tabIndex={0}
            >
              <span
                className="sc-gzVnrw grgfgk"
              >
                Location
              </span>
            </div>
            <input
              id="location-customized-select"
              name="location"
              type="hidden"
              value="Location"
            />
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root MuiSelect-icon"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M7 10l5 5 5-5z"
              />
            </svg>
          </div>
        </div>
        <div
          aria-label="year"
          className="MuiInputBase-root WithStyles(ForwardRef(InputBase))-root-44 MuiInputBase-formControl Footer-select-7"
          onClick={[Function]}
        >
          <div
            className="MuiSelect-root"
          >
            <div
              aria-haspopup="true"
              aria-pressed="false"
              className="MuiSelect-select MuiSelect-selectMenu MuiInputBase-input WithStyles(ForwardRef(InputBase))-input-45"
              id="select-year"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="button"
              tabIndex={0}
            >
              <span
                className="sc-gzVnrw grgfgk"
              >
                Year
              </span>
            </div>
            <input
              id="year-customized-select"
              name="year"
              type="hidden"
              value="Year"
            />
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root MuiSelect-icon"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M7 10l5 5 5-5z"
              />
            </svg>
          </div>
        </div>
      </div>
      <div
        className="MuiFormControl-root Footer-multipleSelectFormControl-3"
      >
        <label
          className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-filled Footer-MultipleSelectInputLabel-6"
          data-shrink={false}
          htmlFor="selectMultipleCheckbox"
        >
          Tags
        </label>
        <div
          aria-label="tags"
          className="MuiInputBase-root MuiInput-root MuiInputBase-formControl MuiInput-formControl"
          onClick={[Function]}
        >
          <div
            className="MuiSelect-root"
          >
            <div
              aria-haspopup="true"
              aria-pressed="false"
              className="MuiSelect-select MuiSelect-selectMenu MuiSelect-filled MuiInputBase-input MuiInput-input"
              onBlur={[Function]}
              onClick={[Function]}
              onFocus={[Function]}
              onKeyDown={[Function]}
              role="button"
              tabIndex={0}
            >
              <span
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "&#8203;",
                  }
                }
              />
            </div>
            <input
              id="selectMultipleCheckbox"
              type="hidden"
              value=""
            />
            <svg
              aria-hidden="true"
              className="MuiSvgIcon-root MuiSelect-icon"
              focusable="false"
              role="presentation"
              viewBox="0 0 24 24"
            >
              <path
                d="M7 10l5 5 5-5z"
              />
            </svg>
          </div>
        </div>
      </div>
    </section>
    <section
      className="sc-EHOje sXSQn"
    >
      <div
        className="MuiFormControl-root Footer-searchFormControl-8"
      >
        <div
          className="MuiFormControl-root MuiTextField-root Footer-searchTextField-9"
        >
          <label
            className="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated Footer-searchTextFieldLabel-10"
            data-shrink={false}
            htmlFor="outlined-search"
          >
            Search...
          </label>
          <div
            className="MuiInputBase-root MuiInput-root MuiInputBase-formControl MuiInput-formControl"
            onClick={[Function]}
          >
            <input
              aria-invalid={false}
              className="MuiInputBase-input MuiInput-input MuiInputBase-inputTypeSearch MuiInput-inputTypeSearch"
              disabled={false}
              id="outlined-search"
              onBlur={[Function]}
              onChange={[Function]}
              onFocus={[Function]}
              required={false}
              type="search"
              value=""
            />
          </div>
        </div>
      </div>
      <section
        className="sc-bZQynM jABVuX"
      >
        <button
          className="MuiButtonBase-root MuiButton-root Footer-searchButton-11 MuiButton-text"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Search
          </span>
        </button>
        <button
          className="MuiButtonBase-root MuiButton-root Footer-resetButton-12 MuiButton-text"
          disabled={false}
          onBlur={[Function]}
          onClick={[Function]}
          onFocus={[Function]}
          onKeyDown={[Function]}
          onKeyUp={[Function]}
          onMouseDown={[Function]}
          onMouseLeave={[Function]}
          onMouseUp={[Function]}
          onTouchEnd={[Function]}
          onTouchMove={[Function]}
          onTouchStart={[Function]}
          tabIndex={0}
          type="button"
        >
          <span
            className="MuiButton-label"
          >
            Reset
          </span>
        </button>
      </section>
    </section>
  </footer>,
  " ",
]
`;
